test/models/medium_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+class MediumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+#validations
+  test 'A Medium Must Have a Title' do
+  	assert media(:flood).valid?
```

Comment:
In order for this to actually test that a medium must have a title, you should also:

1. Remove the title by setting it to nil
1. Assert that the medium object is no longer valid